src/model/CoilHeatingWaterToAirHeatPumpEquationFit.hpp

```diff
@@ -94,6 +95,8 @@ namespace model {
     OS_DEPRECATED(3, 2, 0) double heatingPowerConsumptionCoefficient4() const;
     OS_DEPRECATED(3, 2, 0) double heatingPowerConsumptionCoefficient5() const;
 
+    Curve partLoadFractionCorrelationCurve() const;
```

Comment:
PLF Correlation Curve for a Heating coil. This is not optional. E+ will throw if it's not set. Cf my comments here: https://github.com/NREL/EnergyPlus/pull/10043#pullrequestreview-1602281608

I've made it required-field in both OpenStudio.idd and ProposedEnergy+.idd